zuul.d/fetch-logs.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: all
+  tasks:
+    - name: Output logs
+      shell: "docker-compose logs > /tmp/docker-compose.log 2>&1"
```

Comment:
Suggestion: `gzip` the logs to take less space

Reply:
Yes why not
I'll copy https://github.com/TinxHQ/wazo-production-sf-jobs/blob/master/playbooks/wazo-c4-test/fetch-logs.yaml